GraphSPME/tests/testthat/test_graphspme.R

```diff
@@ -0,0 +1,6 @@
+library(GraphSPME)
```

Comment:
After rebase, the implementation of `cov_ml` is in the .hpp file.
We then include this separately for Python and R packages.
What is a good way to test the C++ implementation?
The "natural" thing I feel is to write tests directly on the C++ side, but that might be painful? At least, I am unsure how to do this.
One way, like here, is to expose the C++ functions on the R and/or Python side and test them against native R/Python functions. This is something that I too know how to do.
What do you think @dafeda ?